### ISSUE TITLE
DRG Wyvern breaths are going through resistance checks

### ISSUE THREAD

**Issue by Steam-arch** [EdenServer/community#6032]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted.
- [x] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [ ] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

DRG pet wyvern breaths should most likely be due to resists, according to wiki 
![image](https://github.com/user-attachments/assets/4e0cd391-25cc-4a52-a92c-fe825b46bfec)
Makes sense because theres AI inplace that checks the weakness of the monster and makes the wyvern plan its breath element according to that.

Eden Khimaira tp wyvern breath spam, all of the breaths have little variance
![image](https://github.com/user-attachments/assets/6b205228-b551-4e5b-b783-7e90e1321fa3)
Same issue was with sky gods.

Senegoid checked the subject also and seems to find that breaths indeed ingore resist checks
![image](https://github.com/user-attachments/assets/631b7495-1d00-4f24-8520-8a0f6161c243)

![image](https://github.com/user-attachments/assets/5eeafe14-d8bc-4744-818c-9cc0d2e7adf6)

**Comment by Sirhc137** [comment]
If those breaths ignored resistance checks, none of those prints would display. You seem to have misunderstood me

**Comment by Steam-arch** [comment]
True! Alright thanks got it now, ill just delete this after, but how are wyvern breaths getting practically zero resist on Sky gods. Assuming khimaira was cosntantly locking it at 1/2

**Comment by Sirhc137** [comment]
Like skillchains, where L2 and L3 skillchains have multiple associated elements, wyvern breaths are programed to select the element the target is weakest to. Where this is deterministic with skillchains, breaths have only a chance to select the 'correct' element. Resistance calculations have high dependence on the natural elemental resistance values (the SDT values) of the target mob.
That might explain why khim is resisting so hard, while the sky gods have weaknesses there.
